### PR TITLE
Add the benchmkar for the inference

### PR DESCRIPTION
**Pull request by libratiger**
## Description
Fix the inference benchmark https://github.com/OpenMined/SyMPC/issues/168

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

File: benchmarks/module/conv_model.py [hunk 1]
```diff
@@ -68,10 +74,23 @@ def run_conv_model(get_clients: Callable[[int], List[Any]]):
...
+    Args:
+        expected: the origin model
+        mpc_model: the Syft model
+        x_mpc: the MPCTensor
+    """
+    """Run the inference on the passed in model"""
+    res_mpc = mpc_model(x_mpc)
     res_mpc.reconstruct()
+
     expected = expected.detach().numpy()
```
Comment (aanurraj) [inline benchmarks/module/conv_model.py:96]: you can remove these, we don't need them 😃

**Review by aanurraj — CHANGES_REQUESTED** [review]
looks great, left a comment

File: benchmarks/module/module_benchmark_test.py [hunk 2]
```diff
@@ -5,14 +5,16 @@
...
+def test_run_inference_conv_model(benchmark, get_clients: Callable[[int], List[Any]]):
+    """Benchmark inference on simple convolutional network.
 
     Arguments:
         benchmark: Fixture that benchmarks any function passed
         get_clients: Fixture that returns a list of clients
     """
-    benchmark(run_conv_model, get_clients)
+    expected, mpc_model, x_mpc = set_up_model(get_clients)
+    benchmark(run_inference_conv_model, expected, mpc_model, x_mpc)
```
Comment (gmuraru) [inline benchmarks/module/module_benchmark_test.py:20]: Q: @aanurraj you said that it might be pretty hard to have 2 plots on the same graph, right?  (I was thinking if we can place the torch convolution with our MPC variant convolution)

Comment (aanurraj) [reply benchmarks/module/module_benchmark_test.py:20]: we can try that, i can check it on my test repo for benchmark action :D

File: benchmarks/module/conv_model.py (hunk 1 above)
Comment (libratiger) [reply benchmarks/module/conv_model.py:96]: @aanurraj is there any todo for us to merge this PR?

Comment (gmuraru) [reply benchmarks/module/conv_model.py:96]: I think it is good to go! I will ping @aanurraj

**Review by aanurraj — APPROVED** [review]
🎉
